CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Bug Fixes
 
+* Fix unnecessary whole project recompilation with static frameworks
+[volodg](https://github.com/volodg)
+[#issue_number](https://github.com/CocoaPods/CocoaPods/issues/issue_number)
```

Comment:
Your issue number is https://github.com/CocoaPods/CocoaPods/issues/7187 - thanks 👍